test/tools/obdscanner.uts

```diff
@@ -259,30 +253,26 @@ s1_pid03 = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID03(fuel_sys
 s1_pid0B = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID0B(data=100)])
 s1_pid0F = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID0F(data=50)])
 s1_queue.put(s1_pid00)
```

Comment:
In my opinion, the design of this tests is not perfect. There are better and more reliable ways to get identical test cases

Reply:
See new commit.